static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -2,6 +2,7 @@
 import { commercialPrebidSafeframe } from 'common/modules/experiments/tests/commercial-prebid-safeframe.js';
 import { commercialAdVerification } from 'common/modules/experiments/tests/commercial-ad-verification.js';
 import { commercialCmpCustomise } from 'common/modules/experiments/tests/commercial-cmp-customise.js';
+import { commercialOutbrainTesting } from 'common/modules/experiments/tests/commercial-outbrain-testing.js';
```

Comment:
Should there be a new file included in this PR called `commercial-outbrain-testing.js` ?

Reply:
Oops! 🙈